### PR TITLE
Remove use of alias

### PR DESCRIPTION
**Pull request by ryandberger**
- Fulfills [STCLI-155](https://issues.folio.org/browse/STCLI-155)
- Deprecated `alias` command and updated documentation in favor of `workspace`
- Note that `alias` was deprecated in the description and that an upcoming release of `yargs` should provide support that our documentation generator can leverage. See https://github.com/yargs/yargs/issues/2115 for details.

### CODE REVIEW

**Review by skomorokh — APPROVED** [review]
Hooray :tada: it's the beginning of the end of this, I presume the idea is you're marking it as a deprecated now in anticipation of removing it with the next major release?

**Review by mkuklis — APPROVED** [review]
Nice cleanup @ryandberger!

File: doc/commands.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Stripes CLI Commands
 
-Version 1.20.0
+Version 2.5.0
```
Comment (MikeTaylor) [inline doc/commands.md:3]: Yikes!!

**Review by MikeTaylor — APPROVED** [review]
Very welcome!

**Comment by ryandberger** [comment]
> Hooray 🎉 it's the beginning of the end of this, I presume the idea is you're marking it as a deprecated now in anticipation of removing it with the next major release?

Yes, seems appropriate to remove with a major release, though I'd want to determine a path for someone to manually remove global aliases when the command no longer exists.

**Pull request merged**